### DISCUSSION TITLE
HA Media Browser Error

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#360] [General]
@blakeblackshear first I really like the HA Integration and thanks again for all your work.
The clips are working and I can play inside the browser but when I choose chromecast as player and click on the clips icon, it shows following error -
![image](https://user-images.githubusercontent.com/7528001/101839094-9fa9c000-3af6-11eb-9356-e175cbf83afa.png)

**Comment by paularmstrong** [comment]
When you choose the player, click on "Local Media > Frigate" instead of the first top-level Frigate and you should be able to use the clips from there.

**Reply by bm55** [reply]
Yes, that is working if I use the "web browser" as a player but if I choose "chromecast" as a player I get the error.

**Reply by paularmstrong** [reply]
Seems to work fine on my chromecast when going to Local Media

**Reply by bm55** [reply]
The local media works with chromecast as player on mine too but the frigate clips shows the error with chromecast player

**Comment by blakeblackshear** [comment]
This is a bug. I opened an issue to fix it.

**Reply by bm55** [reply]
Thanks!
